mmv1/products/certificatemanager/IssuanceConfigs.yaml

```diff
@@ -0,0 +1,127 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'IssuanceConfigs'
+base_url: 'projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs'
+create_url: 'projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs?certificateIssuanceConfigId={{name}}'
+self_link: 'projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs/{{name}}'
+immutable: true
+description: |
+  Certificate represents a HTTP-reachable backend for a Certificate.
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+docs: !ruby/object:Provider::Terraform::Docs
+autogen_async: true
+import_format: ["projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs/{{name}}"]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'certificate_manager_issuance_configs'
+    primary_resource_id: 'default'
+    vars:
+      ca_name: 'my-ca'
+      pool_name: 'my-ca-pool'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    description: |
+      A user-defined name of the certificate issuance config.
+      CertificateIssuanceConfig names must be unique globally
+      and match pattern projects/*/locations/*/certificateIssuanceConfigs/*.
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The Certificate Manager location. If not specified, "global" is used.
+    default_value: global
+    url_param_only: true
+properties:  
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      One or more paragraphs of text description of a CertificateIssuanceConfig.
+  - !ruby/object:Api::Type::Integer
+    name: 'rotationWindowPercentage'
+    description: |
+      It specifies the percentage of elapsed time of the certificate lifetime to wait before renewing the certificate. 
+      Must be a number between 1-99, inclusive. Defaults to 66. 
+      You must set the rotation window percentage in relation to the certificate lifetime so that certificate renewal occurs at least 7 days after 
+      the certificate has been issued and at least 7 days before it expires.
+  - !ruby/object:Api::Type::Enum
+    name: keyAlgorithm
```

Comment:
Same as above- this looks required:

![image](https://github.com/GoogleCloudPlatform/magic-modules/assets/2080520/adf67ffc-cf6c-48be-8545-1600e56e5cbc)